src/commands/dev/dev.js

```diff
@@ -308,6 +308,9 @@ const dev = async (options, command) => {
     const netlifyToken = await command.authenticate()
     await OneGraphCliClient.ensureAppForSite(netlifyToken, site.id)
     const netlifyGraphConfig = await getNetlifyGraphConfig({ command, options })
+
+    log(`Starting Netlify Graph session, to edit your library run \`netlify graph:edit\` in another tab`)
```

Comment:
Will it eventually all run from one command?

Reply:
No, that doesn't make much sense with the current model. All `netlify graph:edit` does is upload your current GraphQL operations library and then open the browser to the url to edit it in the gui.

Down the road there are a lot of improvements we could do by 1. making it offline-capable and 2. moving it inside of vscode via a editor plugin.

Reply:
Also, I punted on it for this iteration since it turned out to be a bit more work than I wanted, but the next iteration could say,

> Starting Netlify Graph session, to edit your library visit <the URL that netlify graph:edit would produce> or run `netlify graph:edit` in another tab